Extensions/shuffle+.js

```diff
@@ -1,148 +1,269 @@
-            return true;
-        },
-        "shuffle"
-    );
-    cntxMenu.register();
+                .popup-row .col {
+                    display: flex;
+                    padding: 10px 0;
+                    align-items: center;
+                }
+                .popup-row .col.description {
+                    float: left;
+                    padding-right: 15px;
+                }
+                .popup-row .col.action {
+                    float: right;
+                    text-align: right;
+                }
+                .popup-row .div-title {
+                    color: var(--spice-text);
+                }                
+                .popup-row .divider {
+                    height: 2px;
+                    border-width: 0;
+                    background-color: var(--spice-button-disabled);
+                }
+                button.checkbox {
+                    align-items: center;
+                    border: 0px;
+                    border-radius: 50%;
+                    background-color: rgba(var(--spice-rgb-shadow), 0.7);
+                    color: var(--spice-text);
+                    cursor: pointer;
+                    display: flex;
+                    margin-inline-start: 12px;
+                    padding: 8px;
+                }
+                button.checkbox.disabled {
+                    color: rgba(var(--spice-rgb-text), 0.3);
+                }
+                select {
+                    color: var(--spice-text);
+                    background: rgba(var(--spice-rgb-shadow), 0.7);
+                    border: 0;
+                    height: 32px;
+                }
+                ::-webkit-scrollbar {
+                    width: 8px;
+                }`
+        );
+
+        function DisplayIcon({ icon, size }) {
+            return React.createElement("svg", {
+                width: size,
+                height: size,
+                viewBox: "0 0 16 16",
+                fill: "currentColor",
+                dangerouslySetInnerHTML: {
+                    __html: icon,
+                },
+            });
+        }
+
+        function checkBoxItem({ name, field, onclickFun = () => {} }) {
+            let [value, setValue] = useState(CONFIG[field]);
+            return React.createElement(
+                "div",
+                { className: "popup-row" },
+                React.createElement("label", { className: "col description" }, name),
+                React.createElement(
+                    "div",
+                    { className: "col action" },
+                    React.createElement(
+                        "button",
+                        {
+                            className: "checkbox" + (value ? "" : " disabled"),
+                            onClick: async () => {
+                                let state = !value;
+                                CONFIG[field] = state;
+                                setValue(state);
```

Comment:
```suggestion
                                CONFIG[field] = !value;
                                setValue(!value);
```
Don't really need a var for this since you've done the same without on other functions also

Reply:
ok